pyproject.toml

```diff
@@ -17,6 +17,7 @@ classifiers = [
     "License :: OSI Approved :: MIT License",
     "Operating System :: OS Independent",
 ]
+dependencies = ["requests~=2.31.0"]
```

Comment:
Apparently there is a way to do this dynamically and reference the requirements file but I couldn't get it to work and don't have the capacity to dive into it right now. We can add it at a later date

Reply:
Adding this to https://privateai.atlassian.net/browse/DEID-2283